docs/Live-main/Song.py

```
@@ -0,0 +1,1725 @@
+        """
+        Continue playing the song from the current position.
+        """
+        return
+
+    def count_in_duration_has_listener(self, arg2: Callable) -> bool:
+        """
+        Returns true, if the given listener function or method is connected to the property "count_in_duration".
+        """
+        return False
+
+    def create_audio_track(self, index: int = None) -> Track:
```

Ruff (0.15.0)

[warning] Unused method argument: `index`

(ARG002)